NEWS.md

```diff
@@ -1,3 +1,7 @@
+# pharmaverseadam 1.0.0
+
+## New Features
```

Comment:
Could we please update here so that it's not completely empty under 1.0.0, otherwise it begs the question as to why we are doing a release at all! Maybe you could replace the "New features" with some other title and write something along the lines of "refreshed dataset with ADaM datasets created by admiral 1.1.1, admiralophtha 1.1, etc..."